modules/machineset-non-guaranteed-instance.adoc

```diff
@@ -3,30 +3,21 @@
 // * machine_management/creating_machinesets/creating-machineset-aws.adoc
 
 [id="machineset-non-guaranteed-instance_{context}"]
-= MachineSet Spot Instances
+= MachineSets that deploy machines as Spot Instances
```

Comment:
Our guidance now is to not pluralize object types. See "object references" in [this](https://docs.google.com/document/d/1SNEFU23Xn6RzGjuZhUe4dMp66otqqHBvaCRyPxuljxk/edit?ts=5f97561e) terminology doc. 
Maybe this should be rewritten as, "MachineSet objects that deploy machines as Spot Instances"? Too bad "MachineSet Spot Instances" doesn't work, cuz it's much easier to read. 😝 

Alternatively, you should be able to write it as, "Machine sets that deploy machines as Spot Instances", right?

Reply:
> Our guidance now is to not pluralize object types. See "object references" in [this](https://docs.google.com/document/d/1SNEFU23Xn6RzGjuZhUe4dMp66otqqHBvaCRyPxuljxk/edit?ts=5f97561e) terminology doc.
> Maybe this should be rewritten as, "MachineSet objects that deploy machines as Spot Instances"? Too bad "MachineSet Spot Instances" doesn't work, cuz it's much easier to read. 😝
> 
> Alternatively, you should be able to write it as, "Machine sets that deploy machines as Spot Instances", right?

Thank you for linking to the terminology doc. This one is tough because the whole assembly uses either MachineSet or MachineSets.